proto/echo/v1/echo.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package echo.v1;
+option go_package = "foxygo.at/pb/echo";
```

Comment:
i dont understand this, sorry. the package is generated to `pb/echo/v1` within the `foxygo.at/pony` module. Should this name have "pony" in it, and what happened to "v1"?

You can just say "trust me, this is right" and I will, but I don't understand it.

Reply:
just outright wrong. sorry. I fixed it.
i've also gotton rid of v1. just in the way for this.